Deploy/stacks/Chile/stack-manager/inputs/data/webspace/A1windpotential.json

```diff
@@ -0,0 +1,137 @@
+{
+    "name": "Chile",
+    "groups": [
+                {
+            "name": "Wind farm suitability Cyan:good, Magenta:bad, Yellow:allowed",
+            "sources": [
+                {
+                    "id": "calc1atlas_capacity_iec1-1",
+                    "type": "raster",
+                    "tiles": [
+                        "http://146.190.86.59:3852/geoserver/ows?service=WMS&version=1.1.0&request=GetMap&layers=the_world_avatar:calc1atlas_capacity_iec1&bbox={bbox-epsg-3857}&width=256&height=256&srs=EPSG:3857&transparent=true&format=image/png"
```

Comment:
Replace 146.190.86.59:3852 with ${IP_ADDRESS}:${PORT} here in all files where it appears.